README.md

```diff
@@ -8,7 +8,7 @@
 [![Coverage Status](https://img.shields.io/coveralls/github/git-chglog/git-chglog.svg?style=flat-square)](https://coveralls.io/github/git-chglog/git-chglog?branch=master)
 [![MIT License](http://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://github.com/git-chglog/git-chglog/blob/master/LICENSE)
 
-> CHANGELOG generator implemented in Go (Golang).  
+> CHANGELOG generator implemented in Go (Golang).
```

Comment:
I would like to keep the line breaks :dog: :pray:
The same goes for the other changes.

Reply:
Ok. Fixed. They where removed automatically by my editor.